content/en/monitors/service_level_objectives/monitor.md

```diff
@@ -6,57 +6,75 @@ further_reading:
+A 99.999% target for a 7-day or 30-day time window results in an error budget of 6 seconds or 26 seconds, respectively. Monitors evaluate every minute, so the granularity of a monitor-based SLO is also 1 minute. Therefore, one alert would fully consume and overspend the 6 second or 26 second error budget in the previous example. In practice, teams cannot satisfy such small error budgets.
 
-Example: `99% of the time requests should have a latency of less than 300ms over the past 7 days`.
+If you need finer granularity than the once a minute monitor evaluation, consider using [metric-based SLOs][3] instead.
 
-While the SLO remains above the target percentage, the SLO's status will be displayed in green font. When the target percentage is violated, the SLO's status will be displayed in red font. You can also optionally include a warning percentage that is greater than the target percentage to indicate when you are approaching an SLO breach. When the warning percentage is violated (but the target percentage is not violated), the SLO status will be displayed in yellow font.
+### Add name and tags
 
-**Note:** The number of decimal places you can specify for your monitor-based SLOs differs depending on the time windows you choose. Up to two decimal places are allowed for 7-day and 30-day targets, and up to three decimal places are allowed for 90-day targets. The precision shown in the details UI of the SLO will be up to `num_target_decimal_places`, meaning two decimal places for 7-day and 30-day targets and three decimal places for 90-day targets. These limits are in place because monitor-based SLO error budget is specified in terms of time. A 99.999% target for a 7-day and 30-day targets results in error budgets of 6 seconds and 26 seconds, respectively. Monitors evaluate every minute, thus the granularity of a monitor-based SLO is also 1 minute. This means one alert would fully consume and overspend the 6 second or 26 second error budgets in the previous examples (for use cases requiring more granularity, consider using [metric-based SLOs][6] instead). Furthermore, error budgets this small are likely unrealistic goals for you to try to achieve in practice.
+Choose a name and extended description for your SLO. Select any tags you would like to associate with your SLO.
 
-### Identify this indicator
+## Example
 
-Here you can add contextual information about the purpose of the SLO, including any related information or resources in the description and tags you would like to associate with the SLO.
+You might be tracking the uptime of a physical device. You have already configured a metric monitor on `host:foo` using a custom metric. This monitor might also ping your on-call team if it's no longer reachable. To avoid burnout, you want to track how often this host is down. A monitor-based SLO using your custom metric can show you at a glance whether the amount of downtime for the `foo` host falls within your target range.
```

Comment:
This paragraph makes it sound like tagging your SLO is unique to custom metrics, which isn't true